packages/renderer/src/components/SceneList/component.tsx

```diff
@@ -129,7 +133,7 @@ export function SceneList({ projects, sortBy, onSort }: Props) {
       </Column>
       <Box
         display="grid"
-        gridTemplateColumns="repeat(4, 1fr)"
+        // gridTemplateColumns="repeat(4, 1fr)"
```

Comment:
is this ok? I see the UI broken if we remove it

Reply:
uh you are right, but this also breaks the empty state. It should be removed only on the empty state but not if you have scenes.